src/transactions.js

```diff
@@ -316,7 +316,11 @@ class Transactions extends Component {
         console.log("RESULT:",result)
         cb(result)
       }
-    })
+    }
+
+    // Promisified versions of tx
+    utils.pTx = async (...args) => new Promise(resolve => utils.tx(...args, resolve))
```

Comment:
No need for `async` keyword. Returning promise is enough

Reply:
https://github.com/leapdao/dapparatus/pull/2/commits/1e468f1a7b6e2b335fbab2c26e51440f674f8aba